### DISCUSSION TITLE
Kitty window layout

### DISCUSSION THREAD

**Discussion by shurik** [kovidgoyal/kitty#5180] [Q&A] [answered]
Hello, I've configured kitty to start with a tall layout and 4 windows. Is it possible to have a keyboard shortcut to toggle the current window to take up the whole screen?

**Comment by kovidgoyal** [comment, marked as answer]
https://sw.kovidgoyal.net/kitty/actions/#action-toggle_layout


On Wed, Jun 08, 2022 at 11:44:59AM -0700, Sasha Vladimirskiy wrote:
> Hello, I've configured kitty to start with a tall layout and 4 windows. Is it possible to have a keyboard shortcut to toggle the current window to take up the whole screen?
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/kovidgoyal/kitty/discussions/5180
> You are receiving this because you are subscribed to this thread.
> 
> Message ID: ***@***.***>

-- 
_____________________________________

Dr. Kovid Goyal 
https://www.kovidgoyal.net
https://calibre-ebook.com
_____________________________________

**Reply by shurik** [reply]
Thank you, sir!
